MarkBind/vue-strap
# Add lint fix script and fix eslint errors

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [ ] Documentation update
• [ ] Bug fix
• [ ] New feature
• [ ] Enhancement to an existing feature
• [x] Other, please explain: Process related: style, linting



Fixes https://github.com/MarkBind/markbind/issues/1213



**What is the rationale for this request?**

In the VueStrap repo, when running `npm run lintwin` there are linting errors. Some are due to CRLF/LF line endings, but there are other errrors, mainly for the rule `import/no-extraneous-dependencies`

**What changes did you make? (Give an overview)**

Added an npm script ` npm run lintfix` to fix linting errors like we have for MarkBind.
Ran it once and included the fixes to the linting errors. 

**Is there anything you'd like reviewers to focus on?**

Due to `import/no-extraneous-dependencies` I had to 
- install `terser-webpack-plugin` 
- shift some modules to `dependencies` from `devDependencies`

**Testing instructions:**

Checkout my branch, run `npm lint` or `npm lintwin`, then run `npm lintfix`,
Then, build vue-strap once and see if anything regresses if you used the built bundle in markbind.

**Proposed commit message: (wrap lines at 72 characters)**





Add lint fix script and fix eslint errors

There are some linting errors, and there is no command to fix lints
Let us add an npm script to fix linting errors and
Include the fixes as well.
